src/main/scala/Main.scala

```diff
@@ -1,8 +1,84 @@
+
+
+import newmodel.Package
+import newmodel._
+
+import scala.collection.mutable.ListBuffer
+
 /**
  * Created by yaroslav on 11.05.2015.
  */
 object Main {
-  def main(args:Array[String]) ={
-    println("hello world")
+
+  object DummyIndex extends Index{
+    val docElements = ListBuffer.empty[DocElement]
+    override def getConversionsFor(id: ID): Seq[ImplicitConversions] = ???
+
+    override def getKnownSubclasses(id: ID): Seq[ID] = ???
+
+    override def getShadowedConversionsFor(id: ID): Seq[ImplicitConversions] = ???
+
+    override def getLinearSupertypesBy(id: ID): Seq[ID] = ???
+
+    override def getCompanionFor(id: ID): ID = ???
+
+    override def getDocElementBy(id: ID): DocElement = docElements.find((x) => x.id == id).get
```

Comment:
The only issue with scala-meta is that a tree cannot serialize. It contains a scratchpad (read scala compiler used to run the scalameta phase). see [this pr](https://github.com/scalameta/scalameta/pull/74/files#diff-440a40121ded85e9baed8bdc9cd8d2bbR77) and [current code](https://github.com/scalameta/scalameta/blob/master/foundation/src/main/scala/org/scalameta/ast/ast.scala#L88). I thin the reason there is a scala compiler in the tree is to perform [sematic operations](https://github.com/scalameta/scalameta/blob/master/scalameta/src/main/scala/scala/meta/semantic/Api.scala). Those operations are essential for documentation, for example [subtype check](https://github.com/scalameta/scalameta/blob/master/scalameta/src/main/scala/scala/meta/semantic/Api.scala#L97). I spoke with @xeno_by about this and he said the end goal is to remove this internalScratchpad, but not now.

To solve this problem, I wrote a [subset of scalameta AST](
https://github.com/metadoc/metadoc/blob/develop/model/src/main/scala/com.scalakata.metadoc/model/Ast.scala#L5)

and a way to fold [from scalameta AST to metadoc AST](https://github.com/metadoc/metadoc/blob/develop/compilerPlugin/src/main/scala/com.scalakata.metadoc.plugin/Metadoc.scala#L52)